.travis.yml

```diff
@@ -17,19 +14,46 @@ env:
     - GDALBUILD=$HOME/gdalbuild
     - PROJBUILD=$HOME/projbuild
     - PROJVERSION="6.0.0"
-    
-  matrix:
-
-    - GDALVERSION="2.2.4"
-    - GDALVERSION="2.3.3"
-    - GDALVERSION="2.4.1"
-    - GDALVERSION="3.0.0"
-    - GDALVERSION="trunk"
 
 matrix:
+  include:
+    - python: "2.7"
+      env:
+      - GDALVERSION="2.2.4"
+
+    - python: "3.5"
+      env:
+      - GDALVERSION="2.2.4"
```

Comment:
@mwtoews we can eliminate these combinations. 1.9.0 will announce deprecation of Python 2.7 and 3.5 support.